example/DSSAT/PETPT.f

```diff
@@ -16,3 +19,7 @@
 ENDIF
 !###  EO = MAX(EO,0.0)   !gives error in DECRAT_C
 EO = MAX(EO,0.0001)
+
+RETURN
```

Comment:
I do not remember if you have to RETURN something in Fortran.

Reply:
yes, it is also in the original file.

Reply:
OK. Nice.